cvat-ui/src/actions/auth-actions.ts

```diff
@@ -100,3 +117,30 @@ export const authorizedAsync = (): ThunkAction => async (dispatch) => {
         dispatch(authActions.authorizeFailed(error));
     }
 };
+
+export const changePasswordAsync = (oldPassword: string, newPassword1: string, newPassword2: string): ThunkAction => async (dispatch) => {
+    dispatch(authActions.changePassword());
+
+    try {
+        await cvat.server.changePassword(oldPassword, newPassword1, newPassword2);
+        dispatch(authActions.changePasswordSuccess());
+    } catch (error) {
+        dispatch(authActions.changePasswordFailed(error));
+    }
+};
+
+export const loadAuthActionsAsync = (): ThunkAction => async (dispatch) => {
+    dispatch(authActions.loadServerAuthActions());
+
+    try {
+        const promises: Promise<boolean>[] = [
+            isReachable(`${cvat.config.backendAPI}/auth/password/change`, 'OPTIONS'),
+        ];
+        const [allowChangePassword] = await Promise.all(promises);
+
+        dispatch(authActions.loadServerAuthActionsSuccess(allowChangePassword));
+
+    } catch (error) {
+        dispatch(authActions.loadServerAuthActionsFailed(error));
+    }
+}
```

Comment:
Why do we need check if server allows to change password? Can it be false?

Reply:
Yes, if LDAP is used for example. This API endpoint is available only for basic authorization.